src/index.scss

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-@import '~styles/normalize';
-@import '~styles/font-face-imports';
+@import './styles/normalize';
```

Comment:
I was going to ask what the "~" to "." reference change means, then I came across the following which indicates that use of @import is deprecated:

https://sass-lang.com/documentation/at-rules/import